src/cm1.F

```diff
@@ -579,8 +578,8 @@ program cm1
       njp1 = nj+1
       nkp1 = nk+1
       nkm1 = nk-1
-      !$acc update device(nip1,njp1,nkp1,nkm1,ni,nj,nk,nparcels, &
```

Comment:
@sjsprecious  nparcels variable is removed because it is not needed to be on the device.  One of my goals in this PR was to only have variables present on the device if it was necessary.  Otherwise, I let the runtime move them over as needed.  I am concerned that having too many replicated variables present on both CPU and GPU will make the code more fragile than necessary.

Reply:
@sjsprecious, I returned the nparcels variable to the GPU.  Instead of updating the device in the cm1.F file, I moved it back to the param() subroutine where it is initially set.